packages/date-time-picker/package.json

```diff
@@ -57,7 +57,16 @@
         "lit-html"
     ],
     "dependencies": {
-        "@spectrum-web-components/input-segments": "^0.0.1"
+        "@spectrum-web-components/calendar": "^0.0.1",
+        "@spectrum-web-components/field-label": "^0.40.0",
```

Comment:
Versions, here, too. The new packages (`@spectrum-web-components/calendar`, etc.) will manage themselves, so no need to worry about them.